libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -504,3 +504,10 @@ singleton AP_Follow method get_target_location_and_velocity boolean Location'Nul
 singleton AP_Follow method get_target_location_and_velocity_ofs boolean Location'Null Vector3f'Null
 singleton AP_Follow method get_target_heading_deg boolean float'Null
 
+include AP_Airspeed/AP_Airspeed.h depends APM_BUILD_TYPE(APM_BUILD_ArduPlane)||APM_BUILD_COPTER_OR_HELI
+singleton AP_Airspeed depends APM_BUILD_TYPE(APM_BUILD_ArduPlane)||APM_BUILD_COPTER_OR_HELI
+singleton AP_Airspeed alias airspeed
+singleton AP_Airspeed method enabled boolean uint8_t 0 AIRSPEED_MAX_SENSORS
```

Comment:
no limit for enabled is best

Reply:
following on from this, I looked at adding the enabled and healthy check in AP_Airspeed as discussed on the call.  I PR'd a quick and dirty solution to that here: #20534
However, as Micheal rightly says, I should be getting airspeed as a pass by reference and returning a bool on healthy.  I started looking into doing this and it is a suprisingly large refactor to do it properly.  Would like to discuss on the dev call before sinking to many hours into it.